### PR TITLE
Feat/testimonials admin

### PR DESCRIPTION
**Pull request by olastor**
#13

### CODE REVIEW

File: src/app/admin/testimonial/testimonial-admin.component.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+  // testimonial data
+  testimonials: Object[];
+  showOnlyUnreviewed: boolean = true;
+
+  constructor(public testimonialService: TestimonialService) {
+    this.loadTestimonials();
+  }
+
+  scroll() {
```
Comment (binarious) [inline src/app/admin/testimonial/testimonial-admin.component.ts]: Please remove.

File: src/app/admin/testimonial/testimonial-admin.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+.testimonial-date {
+  opacity: .5;
+  font-size: 12px;
+}
+
+.testimonial-input {
+  margin-right: 10px;
+}
+
+.emoji-btn {
```
Comment (binarious) [inline src/app/admin/testimonial/testimonial-admin.css]: Have found this button. Please remove the stuff that is not used here.

**Comment by binarious** [comment]
Please fix the build

**Comment by binarious** [comment]
Can you please rebase onto master?

**Comment by binarious** [comment]
Please squash your commits: http://stackoverflow.com/a/25357146

**Comment by binarious** [comment]
Oh and Travis found a problem: https://travis-ci.org/Sirimangalo/meditation-plus-angular/builds/151855968#L2198

**Comment by olastor** [comment]
Closing in favor of #119

**Pull request closed without merging**